coffee-chats/src/main/webapp/src/entity/User.ts

```diff
@@ -0,0 +1,3 @@
+export interface User {
```

Comment:
Unused?

Reply:
For now, yes (it's part of `AuthState` that the backend returns).